crates/ironrdp-connector/src/lib.rs

```diff
@@ -260,17 +261,24 @@ impl fmt::Display for ConnectorErrorKind {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match &self {
             ConnectorErrorKind::Pdu(_) => write!(f, "PDU error"),
-            ConnectorErrorKind::Credssp(_) => write!(f, "CredSSP"),
-            ConnectorErrorKind::Reason(description) => write!(f, "reason: {description}"),
-            ConnectorErrorKind::AccessDenied => write!(f, "access denied"),
-            ConnectorErrorKind::General => write!(f, "general error"),
-            ConnectorErrorKind::Custom => write!(f, "custom error"),
+            ConnectorErrorKind::Credssp(e) => write!(f, "{:?}: {}", e.error_type, e.description),
+            ConnectorErrorKind::Reason(description) => write!(f, "Reason: {description}"),
+            ConnectorErrorKind::AccessDenied => write!(f, "Access Denied"),
+            ConnectorErrorKind::General => write!(f, "General Error"),
+            ConnectorErrorKind::Custom => {
+                write!(f, "Custom Error")?;
```

Comment:
See also [this section](https://github.com/Devolutions/IronRDP/blob/0f238097e0fd9780b017bde473dc495a759906c1/STYLE.md#formatting-of-error-messages) of our style doc about error formatting